tests/chassis_modules_test.py

```diff
@@ -39,6 +40,77 @@
+   Linecard2|Asic0|Ethernet-IB0     3.3.3.4  24:21:24:05:81:f7     1074790404
+   Linecard2|Asic0|Ethernet-IB0   3333::3:4  24:21:24:05:81:f7     1074790405
+Linecard2|Asic1|PortChannel0002    10.0.0.1  26:8b:37:fa:8e:67     1074790406
+Linecard2|Asic1|PortChannel0002     fc00::2  26:8b:37:fa:8e:67     1074790407
+      Linecard4|Asic0|Ethernet5   10.0.0.11  46:c3:71:8c:dd:2d     1074790406
+      Linecard4|Asic0|Ethernet5    fc00::16  46:c3:71:8c:dd:2d     1074790407
+"""
+
+show_chassis_system_neighbors_output_ipv4="""\
+    System Port Interface    Neighbor                MAC    Encap Index
+-------------------------  ----------  -----------------  -------------
+Linecard2|Asic0|Ethernet4    10.0.0.5  b6:8c:4f:18:67:ff     1074790406
+"""
+
+show_chassis_system_neighbors_output_ipv6="""\
+    System Port Interface    Neighbor                MAC    Encap Index
+-------------------------  ----------  -----------------  -------------
+Linecard4|Asic0|Ethernet5    fc00::16  46:c3:71:8c:dd:2d     1074790407
+"""
+
+show_chassis_system_neighbors_output_asic0="""\
+       System Port Interface    Neighbor                MAC    Encap Index
+----------------------------  ----------  -----------------  -------------
+   Linecard2|Asic0|Ethernet4    10.0.0.5  b6:8c:4f:18:67:ff     1074790406
+   Linecard2|Asic0|Ethernet4     fc00::a  b6:8c:4f:18:67:ff     1074790407
+Linecard2|Asic0|Ethernet-IB0     3.3.3.4  24:21:24:05:81:f7     1074790404
+Linecard2|Asic0|Ethernet-IB0   3333::3:4  24:21:24:05:81:f7     1074790405
+   Linecard4|Asic0|Ethernet5   10.0.0.11  46:c3:71:8c:dd:2d     1074790406
+   Linecard4|Asic0|Ethernet5    fc00::16  46:c3:71:8c:dd:2d     1074790407
+"""
+
+show_chassis_system_lags_output="""\
+                System Lag Name    Lag Id    Switch Id                                     Member System Ports
+-------------------------------  --------  -----------  ------------------------------------------------------
+Linecard2|Asic1|PortChannel0002         1            8  Linecard2|Asic1|Ethernet16, Linecard2|Asic1|Ethernet17
+Linecard4|Asic2|PortChannel0001         2           22  Linecard4|Asic2|Ethernet29, Linecard4|Asic2|Ethernet30
+"""
+
+show_chassis_system_lags_output_1="""\
+                System Lag Name    Lag Id    Switch Id                                     Member System Ports
+-------------------------------  --------  -----------  ------------------------------------------------------
+Linecard4|Asic2|PortChannel0001         2           22  Linecard4|Asic2|Ethernet29, Linecard4|Asic2|Ethernet30
+"""
+
+show_chassis_system_lags_output_asic1="""\
+                System Lag Name    Lag Id    Switch Id                                     Member System Ports
+-------------------------------  --------  -----------  ------------------------------------------------------
+Linecard2|Asic1|PortChannel0002         1            8  Linecard2|Asic1|Ethernet16, Linecard2|Asic1|Ethernet17
+"""
+
 class TestChassisModules(object):
```

Comment:
Do we need to add tests for single asic line cards ?

Reply:
Yes. Please see my response to comment on chassis_modules.py:139-142